Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/Batching/Batching.xaml

```diff
@@ -0,0 +1,105 @@
+                        <Run Text="{Binding timeRatio}"/>%
+                    </Span>
+                    faster on average</TextBlock>
+            </StackPanel>
+        </DataTemplate>
+    </Page.Resources>
+
+    <StackPanel>
+        <TextBlock FontWeight="Bold" FontSize="20" Margin="0,40,0,0">Set the BatchSizeOverride property on your LearningModelSessionOptions object</TextBlock>
+        <StackPanel Margin="0,15,0,0" HorizontalAlignment="Left" Width="1000" Background="#F6F6F6" BorderBrush="#dbdbdb" BorderThickness="1" CornerRadius="5" Padding="15,20,0,22">
+            <TextBlock FontSize="16">
+            <Span Foreground="#3562FF">var</Span> <Span Foreground="#1684B2">options</Span> = <Span Foreground="#3562FF">new</Span> <Span Foreground="#1F9D00">LearningModelSessionOptions</Span>();
+            </TextBlock>
+            <TextBlock FontSize="16" Margin="0,10,0,0">
+            <Span Foreground="#1684B2">options</Span>.BatchSizeOverride = <Span Foreground="#086300">50</Span>;
+            </TextBlock>
+            <TextBlock FontSize="16" Margin="0,10,0,0">
+            <Span Foreground="#3562FF">var</Span> <Span Foreground="#1684B2">session</Span> = <Span Foreground="#3562FF">new</Span>
+            <Span Foreground="#1F9D00">LearningModelSession</Span>(<Span Foreground="#1684B2">model</Span>,
+            <Span Foreground="#1684B2">device</Span>, <Span Foreground="#1684B2">options</Span>);
+            </TextBlock>
+        </StackPanel>
+        <TextBlock Margin="0,10,0,0" Width="1000" HorizontalAlignment="Left" TextTrimming="None" Height="20">
+        <Bold>Note:</Bold> In order for batching to work, your model must have a value of -1 or a variable name in the batch dimension.
+        Additionally, the DATA_BATCH Denotation
+        </TextBlock>
+        <TextBlock>
+        must be added on the batch dimension. To edit your model you can use tools like
+        <Hyperlink TextDecorations="None" NavigateUri="https://github.com/Microsoft/Windows-Machine-Learning/tree/master/Tools/WinMLDashboard">
+            WinML Dashboard.
+        </Hyperlink>
+        </TextBlock>
+        <TextBlock FontWeight="Bold" FontSize="20" Margin="0,60,0,0">Compare inference runtimes with and without batching on the CPU and GPU (if available)</TextBlock>
+        <TextBlock FontSize="16" Margin="0,15,0,0">50 images will be passed through SqueezeNet for classification.</TextBlock>
+        <StackPanel Orientation="Horizontal" Margin="0,20,0,0">
+            <TextBlock FontSize="16" Margin="0,5,15,0">Device</TextBlock>
+            <local_controls:DeviceComboBox x:Name="DeviceComboBox" />
+            <TextBlock FontSize="16" Margin="20,5,15,0">Batch Size</TextBlock>
+            <ComboBox x:Name="BatchSizeComboBox" SelectedIndex="3">
+                <x:String>5</x:String>
+                <x:String>10</x:String>
+                <x:String>25</x:String>
+                <x:String>50</x:String>
+            </ComboBox>
```

Comment:
This might be a question for @smk2007, but does this mean the samples gallery does not use MVVM design pattern?